terra/assetlist.json

```diff
@@ -6509,7 +6509,55 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/noble/images/USDCoin.png",
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/noble/images/USDCoin.svg"
         }
+      ],
+      "base": "cw20:terra1m58hc296srr6xygrrfyu6u0e32pl8d459nfs55pmegke8zlv94rqsjz882",
+      "name": "CatWifHat Project",
+      "display": "CatWifHat",
+      "symbol": "CWH",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra/images/cwh.jpg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra/images/cwh.jpg"
+        }
+      ],
+      "base": "cw20:terra10quf590wyr8tuf3pp05hgwm94n3dgfnma377uwaynl3tj47df7nskwdxgs",
+      "name": "Catnip",
+      "display": "Catnip",
+      "symbol": "CNIP",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra/images/cnip.jpg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra/images/cnip.jpg"
+        }
+      ],
+      "base": "cw20:terra1vzhhsddfe558ug43kn5ct5d3aqj48wnvavlaue4d279eltsp9nnql6q549",
+      "name": "Amplified CatWifHat",
+      "display": "Amplified CatWifHat",
+      "symbol": "AMPCWH",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra/images/ampcwh.jpg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra/images/ampcwh.jpg"
+        }
+      ],
+      "base": "cw20:terra1nnmlzhgw92kwyk036r0m3j3u0ydt3enad4vtxlggly886cujq5csrs6k9u",
+      "name": "Amplified Catnip",
+      "display": "Amplified Catnip",
+      "symbol": "AMPNIP",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra/images/ampnip.jpg"
```

Comment:
Also, please ensure all images have a square canvas/shape to them; this one is rectangular.